React/wbooks/src/Home/Components/BookContainer/Components/BookInfo/style.css

```diff
@@ -0,0 +1,3 @@
+.book-image {
+  height: 182px;
+  width: 128px; }
```

Comment:
the closing `}` should be in a new line, and an empty line before the file ends.